crates/services/txpool/src/service.rs

```diff
@@ -264,14 +264,13 @@ where
                         }
                     };
 
-                    if acceptance != GossipsubMessageAcceptance::Ignore {
```

Comment:
By not notifying the ignore status, these messages will be forced to sit in a cache until they expire taking up gossipsub message capacity.